Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.cs

```diff
@@ -40,6 +41,33 @@ protected override void OnApplyTemplate()
             {
                 VisualStateManager.GoToState(this, StateContentExpanded, false);
             }
+
+            var button = (ToggleButton)GetTemplateChild(ExpanderToggleButtonPart);
+
+            if (button != null)
+            {
+                button.KeyDown -= ExpanderToggleButtonPart_KeyDown;
+                button.KeyDown += ExpanderToggleButtonPart_KeyDown;
+            }
+        }
+
+        private void ExpanderToggleButtonPart_KeyDown(object sender, Windows.UI.Xaml.Input.KeyRoutedEventArgs e)
+        {
```

Comment:
Shouldn't this already be handled by the ToggleButton itself? When you hit the spacebar it should check the toggle button which will expand the control